src/api/utils/summary_cache.py

```diff
@@ -292,6 +317,7 @@ def update_summary_cache(project_id, sample_unit=None, skip_test_project=False,
         skip_updates = True
 
     try:
+        sample_unit = sample_unit or None
```

Comment:
If the default is `None`, why is this line necessary?

Reply:
Will change this was a change to support empty string sample_unit to get around the postgres version not support null's as expected with upserts.